cypress/integration/views/common.view.ts

```diff
@@ -0,0 +1 @@
+export const itemsPerPageSelector = 'button[aria-label="Items per page"]';
```

Comment:
What is the use of this new file? Where has it been used and why not in commoncontrols.view.ts?